### PR TITLE
[script.artistslideshow] 2.0.3

### PR DESCRIPTION
**Pull request by pkscout**
v.2.0.3
- removed support for htbackdrops.org (API access no longer available)
- fix for crash when theaudiodb.com doesn't respond with valid JSON

### CODE REVIEW

File: script.artistslideshow/addon.xml [hunk 1]
```diff
@@ -10,6 +10,8 @@
 		<provides>executable</provides>
 	</extension>
 	<extension point="xbmc.addon.metadata">
+		<news>2.0.3
+		- removed support for htbackdrops.org (API access no longer avaialble)</news>
```
Comment (phil65) [inline script.artistslideshow/addon.xml]: typo :)

Comment (pkscout) [inline script.artistslideshow/addon.xml]: That's the new US spelling of available.  '-)  Extra letters make it more better.

Do you want me to resubmit?

Comment (phil65) [inline script.artistslideshow/addon.xml]: You could update the PR by force-pushing.
If it is too much hassle for you then I would also be fine to just keep that typo included. What´s your preference?

**Comment by pkscout** [comment]
I'm fine with the typo. Nobody really reads that anyway. ;)

**Pull request merged**